config/service_providers.yml

```diff
@@ -1292,3 +1292,28 @@ production:
     attribute_bundle:
       - 'email'
     restrict_to_deploy_env: 'prod'
+
+  # HHS - OIG Utils
+  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:hhs:oigutils':
+    friendly_name: 'HHS OIG Utils'
+    agency: 'HHS'
+    agency_id: 19
+    logo: 'hhs_oig.png'
+    cert: 'hhs_oigutils_prod'
+    acs_url: 'https://hhsoigutils.service-now.com/navpage.do'
+    assertion_consumer_logout_service_url: 'https://hhsoigutils.service-now.com/external_logout_complete.do'
+    block_encryption: 'aes256-cbc'
+    return_to_sp_url: 'https://hhsoigutils.service-now.com'
+    ial: 2
```

Comment:
Since this is IAL2, we will need a failure_to_proof_url (See RRB)

Reply:
They don't have a dedicated page, so we'll use the same `return_to_sp_url` for now.